proto/system_state.proto

```diff
@@ -36,11 +36,14 @@ message SystemState {
   // Last state update time
   string system_state_last_update = 9;
 
+  // Authentication state
+  SysAuthState.State authentication_state = 10;
```

Comment:
I would call this a Mode not a State.  It's not something that changes dynamically, rather it's based on the way things are configured. So, I'd replace the uses of "authentication_state" with "authentication_mode".  Does that make sense?

Reply:
Right, updated it to `authentication_mode`.